### ISSUE TITLE
flash assets

### ISSUE THREAD

**Issue by katieannemills** [GRIFFINCollaboration/griffin-website#4]
Do we have the original art assets used to make the flash embeds on the Physics page? Flash is kind of on its way out, and the exact same visualization can be achieved with a gif.

**Comment by AdamGarnsworthy** [comment]
What are you talking about? Which physics page?

**Comment by AdamGarnsworthy** [comment]
Do you mean the photo reel on the Guelph website? I can get updated stuff for you. Do you have all the stuff from my trshare for content?

**Comment by katieannemills** [comment]
http://trshare.triumf.ca/~garns/GRIFFIN/Physics/

There's flash animations of all the decay processes for each page.

**Comment by SmithJK** [comment]
I've never seen this site. Cool.

If we're porting this, there's a slight physics error: the two protons
emitted in two-proton radioactivity are not "only held in the nucleus by
the centrifugal barrier". There's a Coulomb barrier as well. If it were
just a kinematic centrifugal barrier (usually defined as ell-dependent)
standing in the way of two-proton radioactivity, then two-neutron
radioactivity would occur much more often.

**Comment by katieannemills** [comment]
yep, there are other problems too - too many neutrinos in electron capture, not enough nucleons removed from the parent in cluster decay. Let me know any others!

**Comment by katieannemills** [comment]
and spontaneous fission was discovered in 1940, not 38. '38 was neutron induced.

**Comment by r3dunlop** [comment]
Facts are only important if someone checks them :P

**Comment by AdamGarnsworthy** [comment]
Cool, forgot about these. I had them made by the graphic designer who made our GRIFFIN logo. I have them as .swf files (Shockwave Flash file). I also have stills of all of them as a .jpg. I will contact Whitney and see if I can get them as animated gif files.

**Comment by katieannemills** [comment]
OK, if that's easiest - or, if you have stills of each frame, I can make the gifs myself.

**Comment by AdamGarnsworthy** [comment]
I'm told we can convert them by importing into flash and exporting as animated gifs. Do we have flash at TRIUMF? maybe on trwin?

**Comment by katieannemills** [comment]
I've never even attempted to log in to trwin - I can have a look next week when I have access to a windows machine.
